### ISSUE TITLE
Bring back the old respawn point mechanics where you need to sleep, not just right click a bed, to set it.

### ISSUE THREAD

**Issue by K0-RR** [TheRandomLabs/RandomPatches#196]

**Comment by TheRandomLabs** [comment]
Not a bad idea, but it'll be disabled by default.

**Comment by K0-RR** [comment]
Hi, what about something more of a tweak? Like fire arrows set a block they hit on fire or baby zombies burn in daylight? I wanted to ask here first because I don't want to make a feature request of something that doesn't fit into RandomPatches.

**Comment by TheRandomLabs** [comment]
Yeah, that'd be something for RandomTweaks, which I'll get around to porting when I have time. 😉
